src/builder.ts

```diff
@@ -1108,13 +1124,10 @@ export class SchemaBuilder {
       throw new Error(`Missing required "type" field for ${typeConfig.name}.${fieldConfig.name}`)
     }
     const fieldExtension = new NexusFieldExtension(fieldConfig)
+    const nonNullDefault = this.getNonNullDefault(typeConfig, 'output')
     const builderFieldConfig: Omit<NexusGraphQLFieldConfig, 'resolve' | 'subscribe'> = {
       name: fieldConfig.name,
-      type: this.decorateType(
-        this.getOutputType(fieldConfig.type),
-        fieldConfig.list,
-        this.outputNonNull(typeConfig, fieldConfig)
-      ),
```

Comment:
Mentioned in Slack, but similar question to the 2nd arg as a function removal... do we want to detect the presence of `list / nullable` in the existing object and keep the existing logic... making it a type error but a runtime warning for a bit (with some `TODO: remove` in the code so we remember to remove it)?

Reply:
Does this matter less if we're aiming for a 1.0 cut on December 11?

But I agree this would be nice though